src/SelectUsers.vue

```diff
@@ -256,6 +243,59 @@ export default {
 				}
 			})
 		},
+		filterAndFormatUsers(recvUsers) {
+			let users = []
+			// When adding users to a space, show only those users who are not already member of the space
+			if (this.$route.params.group === undefined) {
+				const space = this.$store.state.spaces[this.$route.params.space]
+				users = recvUsers.filter(user => {
+					return (!(user.uid in space.users))
+				}, space)
+			} else {
+				users = recvUsers.filter(user => {
+					return (!(user.groups.includes(this.$route.params.group)))
+				})
+			}
+			// Filters user that are already selected
+			users = users.filter(newUser => {
+				return this.allSelectedUsers.every(user => {
+					return newUser.uid !== user.uid
+				})
+			})
+			return users.map(user => {
+				return {
+					...user,
+					subtitle: user.subtitle ?? '',
+				}
+			})
+
+		},
+		async handleUploadFile(event) {
+			if (event.target.files[0]) {
+				this.isLookingUpUsers = true
+				const bodyFormData = new FormData()
+				const file = event.target.files[0]
+				const space = this.$store.state.spaces[this.$route.params.space]
+				const spaceObj = JSON.stringify(space)
+				bodyFormData.append('file', file)
+				bodyFormData.append('space', spaceObj)
+				try {
+					const users = await this.$store.dispatch('addUsersFromCSV', {
+						formData: bodyFormData,
+					})
+					this.allSelectedUsers = this.filterAndFormatUsers(users)
+				} catch (err) {
+					console.debug(err)
```

Comment:
Can you replace this console.debug by console.error, please ?
One more, can you add additional information, please ?